### PR TITLE
code-server 4.6.1

### PR DESCRIPTION
**Pull request by jsjoeio**
Created with `brew bump-formula-pr` in favor of https://github.com/Homebrew/homebrew-core/pull/109321

### CODE REVIEW

**Comment by jsjoeio** [comment]
```ruby
  Error: An exception occurred within a child process:
    NameError: uninitialized constant Language::Node
```

Hmm... gotta figure this out

File: Formula/code-server.rb [hunk 1]
```diff
@@ -27,7 +28,7 @@ class CodeServer < Formula
 
   def install
     node = Formula["node@16"]
-    system "yarn", "--production", "--frozen-lockfile"
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec), "--unsafe-perm"
     # @parcel/watcher bundles all binaries for other platforms & architectures
     # This deletes the non-matching architecture otherwise brew audit will complain.
     prebuilds = buildpath/"lib/vscode/node_modules/@parcel/watcher/prebuilds"
```
Comment (carlocab) [inline Formula/code-server.rb:34]: Try removing the two `rmtree`s below, including this line.

Comment (jsjoeio) [reply Formula/code-server.rb]: Thank you! I'll try that now.

**Comment by jsjoeio** [comment]
Really not sure on this one. The modules should be there but they're not. Going to see if I can reproduce locally and debug.

**Comment by jsjoeio** [comment]
Roughly speaking, this is what should be happening:
1. code-server is installed from npm globally
2. post installing it should run the [`postinstall` script](https://github.com/coder/code-server/blob/main/package.json#L26)
3. ah... looks like this script assumes `yarn` https://github.com/coder/code-server/blob/main/ci/dev/postinstall.sh#L22 so i imagine that's why it's not working

**Comment by jsjoeio** [comment]
Looks like we need a change in code-server to fix this. Once we do that, I'll update this PR. Relevant changes: https://github.com/coder/code-server/pull/5533

**Pull request closed without merging**